pkg/reconciler/v1alpha1/revision/revision.go

```diff
@@ -119,6 +119,12 @@ func NewController(
 	configMapInformer corev1informers.ConfigMapInformer,
 	buildInformerFactory duck.InformerFactory,
 ) *controller.Impl {
+	transport := http.DefaultTransport
+	if rt, err := newResolverTransport(k8sCertPath); err != nil {
+		opt.Logger.Errorf("Failed to create resolver transport: %v", err)
```

Comment:
I'm not sure if we want this to be fatal or not. I could make NewController return an error but that's a bigger change.